server/jetstream_cluster.go

```diff
@@ -7638,7 +7638,7 @@ func (mset *stream) processClusterStreamInfoRequest(reply string) {
 	sysc.sendInternalMsg(reply, _EMPTY_, nil, si)
 }
 
-const defaultMaxTotalCatchupOutBytes = int64(128 * 1024 * 1024) // 128MB for now, for the total server.
+const defaultMaxTotalCatchupOutBytes = int64(64 * 1024 * 1024) // 64MB for now, for the total server.
```

Comment:
Might just be worth adding some comments around the choice of number here, i.e. if we send a 64MB chunks when catching up, we expect that to take ~0.5s to transfer on an otherwise idle gigabit link with ideal RTT, but that this value might not be ideal on links that have less throughput or higher latency.